public/data/ayushlohmod.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ayush Lohmod",
+  "type": "personal",
+  "bio": "I am Ayush Lohmod, currently pursuing Batchelors of Computer Application from SGT UNIVERSITY. I am a tech enthusiast. I am always open to collaborating on projects and innovative/disruptive ideas",
+  "avatar": "https://github.com/ayushlohmod.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ayushlohmod",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ayushlohmod_",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/ayushlohmod",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Full Stack development",
+      "date": "September 2021",
+      "icon": "NodeJs",
+      "color": "#90ee90",
+      "description": "Started full stack development courrently learning Node Js",
+      "url": "https://ayushlohmod.github.com/oscar"
```

Comment:
There is another _**broken link**_. Please update it with the correct one! 

![image](https://user-images.githubusercontent.com/88786642/177277103-02b65114-1364-4a38-bf04-6761e2e96df3.png)